fine_tune_LLM/requirements.txt

```diff
@@ -0,0 +1 @@
+openai==0.27.2
```

Comment:
note: do need to specify cleanlab and its dependencies in requirements.txt in examples notebooks